protocol/communities/manager.go

```diff
@@ -4013,7 +4013,26 @@ func (m *Manager) AddCommunityToken(token *CommunityToken) (*CommunityToken, err
 		return nil, ErrOrgNotFound
 	}
 
-	token.Base64Image = m.ImageToBase64(token.Base64Image)
+	if croppedImage != nil && croppedImage.ImagePath != "" {
+		identityImages, err := images.GenerateIdentityImages(
+			croppedImage.ImagePath,
+			croppedImage.X,
+			croppedImage.Y,
+			croppedImage.X+croppedImage.Width,
+			croppedImage.Y+croppedImage.Height,
+		)
+		if err != nil {
+			return nil, err
+		}
+
+		base64img, err := images.GetPayloadDataURI(identityImages[0].Payload)
```

Comment:
Just going for the `SmallSize` identity image here, as I think it's enough. Can go large too, if preferred.

Reply:
There is also OpenAndAdjustImage function which takes CroppedImage and do some adjustments.

Reply:
Thanks! Will make use of that one instead.